tvloader/parser2v3/parse_other_license.go

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+package parser2v3
+
+import (
+	"fmt"
+
+	"github.com/spdx/tools-golang/spdx/v2_3"
+)
+
+func (parser *tvParser2_3) parsePairFromOtherLicense2_3(tag string, value string) error {
```

Comment:
@swinslow i'm not as familiar with the license change assertion of XML format in 2.3.. From my knowledge its just additional validation and parsing required (which we should add to as part of https://github.com/spdx/tools-golang/issues/159)? Thoughts here?